config/environments/production.rb

```diff
@@ -63,10 +63,6 @@
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
 
-  # Use a real queuing backend for Active Job (and separate queues per environment)
```

Comment:
Please don't remove this, unless the lines in `config/application.rb` this actually gets added when scaffolding with `rails new`, so I'd rather leave it in.

Reply:
Sure, we can leave it